### PR TITLE
fix Keypair Base64 Serde

### PR DESCRIPTION
**Pull request by punwai**

### CODE REVIEW

File: crates/sui-benchmark/Cargo.toml [hunk 1]
```diff
@@ -35,7 +35,7 @@ sui-config = { path = "../sui-config" }
 sui-types = { path = "../sui-types" }
 
 move-core-types = { git = "https://github.com/move-language/move", rev = "7733658048a8bc80e9ba415b8c99aed9234eaa5f", features = ["address20"] }
-narwhal-node = { git = "https://github.com/MystenLabs/narwhal", rev = "f50af9eb40130fb404a678e38d0671fc33f60318", package = "node" }
+narwhal-node = { git = "https://github.com/punwai/narwhal-1", rev = "172cf0e36a9a1b357cb31df645086de03deffecf", package = "node" }
```
Comment (bmwill) [inline crates/sui-benchmark/Cargo.toml:38]: Lets not do this and instead land the changes in the narwhal repo and use a commit from the narwhal repo.

**Review by bmwill — CHANGES_REQUESTED** [review]

File: crates/sui-benchmark/Cargo.toml (hunk 1 above)
Comment (punwai) [reply crates/sui-benchmark/Cargo.toml:38]: Yep, I did this to test, the PR should merge in a few minutes.

Comment (bmwill) [reply crates/sui-benchmark/Cargo.toml:38]: ah didn't realize that was the case :)

**Review by bmwill — APPROVED** [review]

**Comment by kchalkias** [comment]
Great, thanks @punwai and @bmwill!

**Pull request merged**